test/smoke_test.dart

```diff
@@ -35,6 +35,7 @@ void main() {
     var tapped = false;
 
     final tapableWidgets = <Widget>[
```

Comment:
@awhitford, new smoke tests should also be written for the `v3.0.0` feature upgrades -- we will have to change the current tests once we remove the redundant code. Opening a new issue will be fine for now.

Reply:
Please note that the `example` code was rewritten, and that is part of the `smoke_test`, so the _new V3 approach_ is being tested too.

Reply:
Got it, thanks!